src/cosalib/clouds/azure.py

```diff
@@ -0,0 +1,7 @@
+from cosalib.cmdlib import run_verbose
+
+
+def remove_azure_image(azure_id):
+    print(f"Azure: removing image {azure_id}")
+    run_verbose(['ore', 'azure', '--log-level', 'debug',
+                 'unreplicate-image', azure_id])
```

Comment:
I'm not positive but I don't believe that `unreplicate-image` is the same as deleting the blob. A quick glance shows that we don't have a `delete-blob-arm` style command in `ore` at the moment, do you want to either implement that or file a ticket and I'll try to get to it soon?

Reply:
Sure, I can work on that - filed https://github.com/coreos/mantle/issues/1115